com.woltlab.wcf/update_5.3.sql

```diff
@@ -0,0 +1,16 @@
+DELETE FROM wcf1_style_variable WHERE variableName = 'useGoogleFont';
+
+DELETE FROM wcf1_package_update_server WHERE (serverURL LIKE '%//update.woltlab.com%' OR serverURL LIKE '%//store.woltlab.com%') AND serverURL NOT LIKE '%/5.3%';
+
+DROP TABLE IF EXISTS wcf1_devtools_missing_language_item;
+CREATE TABLE wcf1_devtools_missing_language_item (
+	itemID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	languageID INT(10),
+	languageItem VARCHAR(191) NOT NULL,
+	lastTime INT(10) NOT NULL,
+	stackTrace MEDIUMTEXT NOT NULL,
+	
+	UNIQUE KEY (languageID, languageItem)
+);
```

Comment:
Is there a reason this can't be done with the update script?

Reply:
I thought so first, but I didn't read the exception carefully and forgot reverting that change.